### DISCUSSION TITLE
How does `signJSON` work?

### DISCUSSION THREAD

**Discussion by July-Only** [trustwallet/wallet-core#2310] [Q&A]
// Signs a json transaction with private key.

I see that there is no use example and explanation of this method in your application document. I hope you can give some explanation and example of ETH. Thank you very much! The method is as follows:
public static func signJSON(_ json: String, key: Data, coin: CoinType)  -> String

**Comment by optout21** [comment]
Maybe this helps?
https://github.com/trustwallet/wallet-core/blob/master/tests/Ethereum/TWAnySignerTests.cpp#L302
